cli/src/code_coverage/prvsnr-coverage.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+
+import argparse
+import subprocess
+import sys
+import os.path
+import shlex
+
+def _run_command(cmd):
+    try:
+        res = subprocess.run(shlex.split(cmd), shell=False)
```

Comment:
Also, would we need `sudo` ever?
I recently discovered subprocess is potentially risky.
Add a check for allowed commands only to be executed, so that no one can attempt:
`rm -rf`

Similar has been attempted in:
https://github.com/Seagate/cortx-prvsnr/blob/150ca3047a25cebbccc79e8cfdbb62f72e3873c3/srv/_modules/setup_conf.py#L72

Reply:
Addressed